www/loginform.php

```diff
@@ -109,11 +109,15 @@
             /* In case of challenge response we do not change the username */
             $state['forcedUsername'] = $username;
             $transaction_id = $errorParams[1];
-            $message = $errorParams[2];
-            $attributes = $errorParams[3];
+            $message = 'Please confirm with one of these tokens: ';
```

Comment:
Can we do i18n here?

Reply:
Fixed here https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50/commits/8050b183489e4d82439288f375fb7efc02b2775c

Reply:
:+1: